src/components/Suggestions.test.tsx

```diff
@@ -0,0 +1,145 @@
+import { CodeEditorSuggestionItemKind } from '@grafana/ui';
+
+import { getSuggestions } from './Suggestions';
+
+const templateSrv = {
+  getVariables: jest.fn().mockReturnValue([{ name: 'foo' }, { name: 'bar' }]),
+  replace: jest.fn(),
+};
+
+jest.mock('@grafana/runtime', () => {
+  return {
+    ...(jest.requireActual('@grafana/runtime') as any),
+    getTemplateSrv: () => templateSrv,
+  };
+});
+
+describe('getSuggestions', () => {
+  it('should include macros', () => {
```

Comment:
What would you think about adding under what circumstances macros should be included? I could see this test breaking in the future after a code change and it being hard to for the future dev to guess what this input represents for a user `[], [], [], [], '', '', ''`

Reply:
true, these inputs are not very readable, I'm going to make these an object